src/libraries/Microsoft.PowerFx.Core/IR/Nodes/BinaryOpKind.cs

```diff
@@ -5,6 +5,8 @@ namespace Microsoft.PowerFx.Core.IR.Nodes
 {
     internal enum BinaryOpKind
     {
+        Invalid,
```

Comment:
Are we ever serializing these or otherwise using the ordinal values of this enum?